### PR TITLE
add commitTS in column for restore

### PR DESCRIPTION
**Pull request by lichunzhu**
### What problem does this PR solve? 
supplement of https://github.com/pingcap/tidb-operator/pull/2899, add commitTS info in restore

### What is changed and how does it work?
add commitTS column, append `backupErr` when `setTiKVGCTime` is failed to avoid err lost.

### Check List 

Tests 

 - Unit test
 - E2E test
 - Stability test
 - Manual test (add detailed scripts or steps below)
test get restore and backup
```bash
$ kubectl get backup basic-backup -n demo  
NAME           BACKUPPATH                                     BACKUPSIZE   COMMITTS             AGE
basic-backup   s3://chunzhu/backup-2020-07-14T10:21:45Z.tgz   3.9 kB       418046831785148417   71m
$ kubectl get restore basic-restore -n demo
NAME            STARTED                COMPLETED              COMMITTS             AGE
basic-restore   2020-07-14T10:23:30Z   2020-07-14T10:23:33Z   418046831785148417   70m
```

Code changes

 - Has Go code change

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
add commitTS info column in `get restore` command
```

### CODE REVIEW

File: cmd/backup-manager/app/backup/manager.go [hunk 1]
```diff
@@ -252,6 +252,9 @@ func (bm *Manager) performBackup(backup *v1alpha1.Backup, db *sql.DB) error {
 	if oldTikvGCTimeDuration < tikvGCTimeDuration {
 		err = bm.SetTikvGCLifeTime(db, oldTikvGCTime)
 		if err != nil {
+			if backupErr != nil {
```
Comment (DanielZhangQD) [inline cmd/backup-manager/app/backup/manager.go:255]: Please also update restore/manager.go

Comment (lichunzhu) [reply cmd/backup-manager/app/backup/manager.go:255]: updated in https://github.com/pingcap/tidb-operator/pull/2926/commits/eaf73927bbd9f066362febba1db3b582f87d0444

**Pull request merged**